jdbc/src/test/java/org/apache/metamodel/jdbc/HsqldbTest.java

```diff
@@ -417,4 +418,98 @@ public void testInterpretationOfNull() throws Exception {
         Connection conn = DriverManager.getConnection("jdbc:hsqldb:mem:interpretation_of_null", USERNAME, PASSWORD);
         JdbcTestTemplates.interpretationOfNulls(conn);
     }
+    
+    public void testCapitalization() throws Exception{
+      try(Connection connection = DriverManager.getConnection("jdbc:hsqldb:mem:different_types_insert", USERNAME,PASSWORD);){
+        
+
+        
+        final JdbcDataContext dcon = new JdbcDataContext(connection);
+        
+        final String mixedcap = "MixedCapitalization";
+        final String idCol = "Id";
+        final String nameCol =  "name";
+        final String emailCol = "EMAIL";
+        final String createTable = "CREATE TABLE \"" + mixedcap + "\" (\""+idCol+"\" INTEGER, \""+nameCol+"\" LONGVARCHAR,  \""+emailCol+"\" LONGVARCHAR)";
+        
+        try(Statement stmt = connection.createStatement();){
+          
+          stmt.execute(createTable);
+          
+        }
+        dcon.refreshSchemas();
+        assertEquals(mixedcap, dcon.getDefaultSchema().getTable(0).getName()); 
+        assertEquals(idCol, dcon.getDefaultSchema().getTable(0).getColumn(0).getName());
+        
+        dcon.query().from(mixedcap).select(idCol,nameCol,emailCol).execute();
+        
+//        Query query = new Query();
+//        String subselect = "(select * from \""+mixedcap+"\")";
+//        query.from(subselect);
+//        query.select("test");
+//        
+//        dcon.executeQuery(query);
```

Comment:
removed it.

This code should demonstrate some other issue i had, but it should better be placed in an other test